.eslintrc.cjs

```diff
@@ -34,6 +35,28 @@ const config = {
     },
   },
   rules: {
+    '@cspell/spellchecker': [
+      'error',
+      {
+        cspell: {
+          words: [
+            'tanstack', // Our package scope
+            'tsqd', // Our public interface (TanStack Query Devtools shorthand)
+            'retryer', // Our public interface
+            'Promisable', // Our public interface
+            'extralight', // Our public interface
+            'codemod', // We support our codemod
+
+            'TSES', // @typescript-eslint package's interface
+            'tsup', // We use tsup as builder
+            'solidjs', // Our target framework
+            'tabular-nums', // https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-numeric
+            'todos', // Too general word to be caught as error
+            'ouuid', // codemod's variable
```

Comment:
this is just a variable in a codemod's example input / output. We can rename this to whatever we want :)

Reply:
@TkDodo I removed this word and changed variable name (ouuid -> id) in [36aa021](https://github.com/TanStack/query/pull/6718/commits/36aa02197bfbbb8c6ce2dcc8c8fa971e516a21a1)